manifests/golang.yml

```diff
@@ -449,7 +449,7 @@ tests/:
       Test_Config_RateLimit: v1.67.0
       Test_Config_TraceAgentURL: v1.68.0
       Test_Config_TraceEnabled: v1.67.0
-      Test_Config_TraceLogDirectory: missing_feature
+      Test_Config_TraceLogDirectory: v1.70.0
```

Comment:
The test is not executed for dev version (1.70.0-dev)
In the restults report:
```   {
      "path": "tests/parametric/test_config_consistency.py::Test_Config_TraceLogDirectory::test_trace_log_directory_configured_with_existing_directory[library_env0]",
      "lineNumber": 53,
      "outcome": "xpassed",
      "testDeclaration": "notImplemented",
      "details": "missing_feature for golang: declared released version is v1.70.0, tested version is 1.70.0-dev",
      "features": [
        325
      ]
    },```

Reply:
@robertomonteromiguel , I'm not sure I'm following; the results seems to say it is testing v1.70.0-dev: "tested version _is_ 1.70.0-dev"
The feature corresponding to this test Test_Config_TraceLogDirectory will be included in next dd-trace-go release v1.70.0; What changes would you like me to make?
Note: My mistake, I meant to write, "Or v1.70.0-dev?" instead of "Or v1.69.0-dev?"

Reply:
1.70.0-dev